src/site/content/en/fast/image-cdns/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: Use image CDNs to optimize images
+authors:
+  - katiehempenius
+description: |
+  Image CDNs are excellent at optimizing images. 
+  Switching to an image CDN can yield a 40–80% savings in image bytes.
+date: 2019-07-28
```

Comment:
mind adding a `draft: true` flag to this post? That way we can land it but hold off on launching it until all of the image content is in place.

Reply:
actually I can push to this branch! I'll add it :D

Reply:
oh snap you beat me to it!